src/audio/dai.c

```diff
@@ -10,6 +10,9 @@
 #include <sof/audio/format.h>
 #include <sof/audio/pcm_converter.h>
 #include <sof/audio/pipeline.h>
+#if CONFIG_CAVS_ALH
```

Comment:
should be next to other `sof/drivers/` includes, we sort them by `visibility` and then `alphabetically`

Reply:
thanks, I will refine it

Reply:
this also begs the question if we need to only conditionally include stuff?